pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -749,6 +751,9 @@ def _read_configuration():
     if args.meta_data is None:
         args.meta_data = []
 
+    if args.android_used_libs is None:
+        args.android_used_libs = []
```

Comment:
Very minor thing: is it enough to just write `default=[]` in the `ap.add_argument(...)` instead?

Reply:
I think it should be, I just wanted it to be consistent with existing code. If you like it I can refactor rest of it to use defaults.

Reply:
fixed